receiver/mongodbreceiver/testdata/integration/scripts/lpu.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+set -e
+
+USER="otelu"
+PASS="otelp"
+MONGO_INITDB_ROOT_USERNAME="otel"
+MONGO_INITDB_ROOT_PASSWORD="otel"
+
+setup_permissions() {
+  mongo -u $MONGO_INITDB_ROOT_USERNAME -p $MONGO_INITDB_ROOT_PASSWORD<<EOF
+  use admin
+  db.createUser(
+          {
+              user: "${USER}",
+              pwd: "${PASS}",
+              roles: [
+                  {
+                      role: "clusterMonitor",
+                      db: "admin"
```

Comment:
Is this testing that a non-admin user can monitor the admin database? If so, seems like an unusual test to exemplify typical behavior.

Reply:
Removed db: "admin" line.